services/backend/src/service/connectivity.js

```diff
@@ -34,15 +38,21 @@ module.exports = () => {
     getPADPC: async () => connectivityDPC,
 
     /**
-     * Get the layers of the protected areas with higher dPC value in a given area
+     * Get the layers of the protected areas with higher dPC value in a given area. If paNumber
+     * is not provided, all layers are returned
      *
      * @param {String | Number} areaType area type
```

Comment:
Check again the param types of areaType and areaId

Reply:
Done